acqa-repo1-aws-tf12-part1.tf

```diff
@@ -458,10 +485,10 @@ resource "aws_autoscaling_group" "acqa-test-asg1" {
   vpc_zone_identifier       = [aws_subnet.acqa-test-albsubnet1.id, aws_subnet.acqa-test-albsubnet2.id]
 
   initial_lifecycle_hook {
-    name                 = "aqa-test-asg1-lifecyclehook1"
-    default_result       = "CONTINUE"
-    heartbeat_timeout    = 2000
-    lifecycle_transition = "autoscaling:EC2_INSTANCE_LAUNCHING"
+    name                  = "aqa-test-asg1-lifecyclehook1"
+    default_result        = "CONTINUE"
+    heartbeat_timeout     = 2000
+    lifecycle_transition  = "autoscaling:EC2_INSTANCE_LAUNCHING"
     notification_metadata = <<EOF
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure KMS customer managed keys are used for server side encryption (SSE) of AWS DyanamoDB tables**
 
**Vulnerability**
AWS DynamoDb tables do not use KMS customer managed keys for server side encryption (SSE).
 
**Remediation**
Enable Server Side Encryption for DyanamoDB with Managed Customer Master Keys. Make sure enabled is set to true and kms_key_arn is set to correct value in server_side_encryption.

source: accurics-